src/core/http-request/__tests__/helpers.spec.ts

```diff
@@ -30,8 +30,8 @@ describe("HttpRequestImpl helpers", () => {
   test("req should instantiate a StringBody when a string is passed as body", () => {
     const expectedBody = stringBody("some-body");
 
-    const request1 = req({ url: "/", body: "some-body" });
-    const request2 = req({ url: "/", body: stringBody("some-body") });
+    const request1 = req("/", HttpMethods.GET, "some-body");
+    const request2 = req("/", HttpMethods.GET, stringBody("some-body"));
```

Comment:
We have to provide default arguments if the needed argument is not the next one. So we should provide the default method because we want to reach the third parameter (body).

We can provide `undefined` but IMO it is not better:
```ts
const request1 = req("/", undefined, "some-body");
```